var/spack/repos/jcsda-emc/packages/mapl/package.py

```diff
@@ -47,6 +48,7 @@ class Mapl(CMakePackage):
     variant('esma_gfe_namespace', default=True)
     variant('shared', default=True)
     variant('debug', default=False, description='Make a debuggable version of the library')
+    variant('extdata2g', default=True, description='Use ExtData2G')
```

Comment:
I missed the conversation on hpc-stack, what does this thing do?

Reply:
Whether or not to use the new ExtData (2G - 2nd Generation) component in MAPL for GOCART. This option was disabled by default, but now it's enabled.